app/lib/constants/settings/general.rb

```diff
@@ -5,6 +5,11 @@ module General
       SVG_PLACEHOLDER = "<svg ...></svg>".freeze
 
       DETAILS = {
+        contact_email: {
+          description: "Used for contact links. Please provide an email address where users " \
+                       "can get in touch with you or your team.",
+          placeholder: "yo@dev.to"
```

Comment:
Since we're changing this already, maybe we shouldn't make placeholders, etc. DEV-specific anymore?

```suggestion
          placeholder: "hello@example.com"
```